### PR TITLE
Improve getting started experience

### PR DESCRIPTION
**Pull request by inlined**
1. Tells customers they can ^C once the rollout has been created
2. Turns the rollout in progress message to a spinny
3. Adds a spinny to wait on TLS to be ready. To be forwards compatible with the day that we can just serve all default domains on a wildcard cert, this dialog only pops up after an initial check verifies that TLS is not yet ready.

Not sure about some of the prose I've moved about. Feel encouraged to nitpick and/or add other nitpickers as reviewers.

### CODE REVIEW

**Comment by inlined** [comment]
Reduced scope to just awaiting TLS now that other changes have been implemented. Changed the rollout text a bit since we may to continue waiting for TLS. Also, I removed a "..." because it seems redundant/passive aggressive along with a spinner.

File: src/apphosting/index.ts [hunk 1]
```diff
@@ -121,6 +148,7 @@ export async function doSetup(
   });
 
   await setDefaultTrafficPolicy(projectId, location, backendId, branch);
+  logSuccess(`Successfully created backend:\n\t${backend.name}`);
```
Comment (tonyjhuang) [inline src/apphosting/index.ts:151]: Remove, this is already on L138

File: src/apphosting/index.ts [hunk 2]
```diff
@@ -147,7 +178,13 @@ export async function doSetup(
       },
     },
   });
-  createRolloutSpinner.succeed(`Your backend is now deployed at:\n\thttps://${backend.uri}`);
+  createRolloutSpinner.succeed("Rollout complete");
+  if (!(await tlsReady(url))) {
+    const tlsSpinner = ora("Waiting for TLS certificate");
```
Comment (tonyjhuang) [inline src/apphosting/index.ts:183]: you probably want tlsSpinner = ora(...).start()

File: src/apphosting/index.ts [hunk 3]
```diff
@@ -134,11 +162,14 @@ export async function doSetup(
...
 
+  const url = `https://${backend.uri}`;
   logBullet(
     `You may also track this rollout at:\n\t${consoleOrigin()}/project/${projectId}/apphosting`,
   );
+  // TODO: Previous versions of this command printed the URL before the rollout started so that
+  // if a user does exit they will know where to go later. Should this be re-added?
   const createRolloutSpinner = ora(
-    "Starting a new rollout... This make take a few minutes. It's safe to exit now.",
+    "Starting a new rollout; this make take a few minutes. It's safe to exit now.",
```
Comment (tonyjhuang) [inline src/apphosting/index.ts:172]: typo: this may take

File: src/apphosting/index.ts (hunk 2 above)
Comment (tonyjhuang) [inline src/apphosting/index.ts:183]: "Finalizing your backend's TLS certificate; this may take a few minutes."

**Review by tonyjhuang — APPROVED** [review]
Left some nits

File: src/apphosting/index.ts (hunk 1 above)
Comment (inlined) [reply src/apphosting/index.ts]: ugh. merge issues. I had previously moved and now it got turned into an add. TY for the catch.

File: src/apphosting/index.ts (hunk 2 above)
Comment (inlined) [reply src/apphosting/index.ts]: Whoops! Maybe Ora isn't as easy to use as I thought!

**Pull request merged**